movierental.jpa/src/main/resources/data.sql

```diff
@@ -29,6 +29,7 @@ INSERT INTO USERS (USER_ID, USER_NAME, USER_FIRSTNAME, USER_EMAIL) VALUES (1, 'K
 INSERT INTO USERS (USER_ID, USER_NAME, USER_FIRSTNAME, USER_EMAIL) VALUES (2, 'Knecht', 'Werner', 'werner.knecht@gmail.com');
 INSERT INTO USERS (USER_ID, USER_NAME, USER_FIRSTNAME, USER_EMAIL) VALUES (3, 'Meyer', 'Barbara', 'barbara.meyer@gmail.com');
 INSERT INTO USERS (USER_ID, USER_NAME, USER_FIRSTNAME, USER_EMAIL) VALUES (4, 'Kummer', 'Adolf', 'adolf.kummer@gmail.com');
+INSERT INTO USERS (USER_ID, USER_NAME, USER_FIRSTNAME, USER_EMAIL) VALUES (5, 'Naef', 'Livio', 'livio.naef@students.fhnw.ch');
```

Comment:
Sie haben hier einen zusätzlichen User in die Daten eingefügt welche auch für die Tests verwendet werden, folglich schlägt nun der Test `testFindAll` in der Klasse `UserRepositoryTest` fehl.